8510Final-AR.qmd

```diff
@@ -0,0 +1,311 @@
+---
+title: "Final Project"
+author: "Megha Khanna"
+date: "05/01/2024"
+---
+# Title of paper here
```

Comment:
Give your paper a title - notice the markdown syntax here. More generally your entire paper wasn't formatted in proper markdown. Please review markdown syntax: https://www.markdownguide.org/